src/scripts/repl_server.py

```diff
@@ -44,15 +51,16 @@
                 excs = __traceback.format_exception_only(e.__class__, e)
             __exc = ''.join(excs).rstrip()
             __traceback.clear_frames(e.__traceback__)
-            __client_socket.send('[Initialize]'.encode())
+            __buf.append('[Initialize]')
         __out = __sys.stdout.getvalue()[:-1]
         # assert not(__exc and __res)
         if __exc or __res:
             __out += '\n'
```

Comment:
By the way, is the following format expected?
![2023-04-22_23-24](https://user-images.githubusercontent.com/9482395/233790507-e230b48d-e87f-40fd-b4fa-a523961a426d.png)

Reply:
Oh, you're right. When `__out` is empty, this line break is unnecessary.

(It is possible for `__out` and `__exc` to be filled at the same time. e.g. `print!("a"); assert 1 == 2`)